[Hotfix]: UserActivityMapper mapping 오류

### Branch Keyword

useractivity-hotfix

### 버그 내용

- UserActivityMapper mapping 오류

### 발생 위치

UserActivityMapper

### 스크린샷

_No response_

### 로그 또는 에러 메시지

/home/runner/work/sb01-monew-team2/sb01-monew-team2/monew/src/main/java/com/codeit/team2/monew/module/domain/useractivity/mapper/UserActivityMapper.java:28: error: Can't map property "Set<InterestKeyword> subscription.interest.keywords" to "List<String> interestKeywords". Consider to declare/implement a mapping method: "List<String> map(Set<InterestKeyword> value)".
    SubscriptionItem toSubscriptionItem(Subscription subscription);

### 관련 API

_No response_

### 테스트 방법

- [ ] 단위 테스트
- [ ] 통합 테스트
- [ ] API 테스트

--- a/monew/src/main/java/com/codeit/team2/monew/module/domain/useractivity/mapper/UserActivityMapper.java
+++ b/monew/src/main/java/com/codeit/team2/monew/module/domain/useractivity/mapper/UserActivityMapper.java
@@ -12,6 +12,7 @@ import com.codeit.team2.monew.module.domain.useractivity.dto.SubscriptionItem;
 import com.codeit.team2.monew.module.domain.useractivity.dto.UserActivityDto;
 import java.util.Collections;
 import java.util.List;
+import java.util.Set;
 import java.util.stream.Collectors;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
@@ -27,7 +28,7 @@ public interface UserActivityMapper {
     @Mapping(source = "interest.keywords", target = "interestKeywords")
     SubscriptionItem toSubscriptionItem(Subscription subscription);
 
-    default List<String> mapInterestKeywords(List<InterestKeyword> keywords) {
+    default List<String> mapInterestKeywords(Set<InterestKeyword> keywords) {
         if (keywords == null) {
             return Collections.emptyList();
         }

--- a/monew/src/test/java/com/codeit/team2/monew/module/domain/useractivity/service/UserActivityServiceImplTest.java
+++ b/monew/src/test/java/com/codeit/team2/monew/module/domain/useractivity/service/UserActivityServiceImplTest.java
@@ -23,6 +23,7 @@ import com.codeit.team2.monew.module.domain.useractivity.mapper.UserActivityMapp
 import java.time.Instant;
 import java.util.List;
 import java.util.Optional;
+import java.util.Set;
 import java.util.UUID;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
@@ -85,7 +86,7 @@ class UserActivityServiceImplTest {
         when(mockInterest.getId()).thenReturn(UUID.randomUUID());
         when(mockInterest.getName()).thenReturn("경제");
         when(mockInterest.getSubscriberCount()).thenReturn(20);
-        when(mockInterest.getKeywords()).thenReturn(List.of(mockInterestKeyword));
+        when(mockInterest.getKeywords()).thenReturn(Set.of(mockInterestKeyword));
         when(mockInterestKeyword.getKeyword()).thenReturn(mockKeyword);
         when(mockKeyword.getName()).thenReturn("금리");
 

